docs/source/system_replication/system_replication.rst

```diff
@@ -0,0 +1,114 @@
+==================================================
+USB-Based System Replication
+==================================================
+
+.. contents:: Table of Contents
+   :depth: 2
+   :local:
+
+Introduction
+============
+
+The `USB-based system replication` feature allows saving the full disk image of an x64
+based NI Linux RT capable target and applying it to same or different target(s) of the
+same model.
+
+Safemode, runmode contents, drivers, can all be replicated.
+
+Some settings such as Hostname, SSH keys, Salt Minion ID, that uniquely identify a
+device on a network are not replicated to prevent conflicts on the network.
+
+Creating USB drive
+==================
+
+NI Linux RT System Image ships with a `nilrt-recovery-media-x64.iso`. Version 24.8 and
+above can be used to create a USB drive capable of system replication.
+
+Format a USB drive with **FAT32** File System and set **NIRECOVERY** as Volume label.
+
+   .. image:: media/format_usb.png
+
+Mount `nilrt-recovery-media-x64.iso` found under
+`C:\\Program Files (x86)\\National Instruments\\RT Images\\Utilities\\Linux RT Safemode\\<version 24.8 and above>`
+and copy the contents to the USB drive.
```

Comment:
Can we reformat this a little so this step is more prominent? It kind of gets lost visually after the large image above. Maybe shrink the image? Or clearly enumerate the steps in a numbered list or something? Or show the steps first and place the image below the steps?

Also, would this wording be better?
"Mount nilrt-recovery-media-x64.iso and copy the contents to the USB drive. Do **not** write the entire iso file to the USB drive. The nilrt-recovery-media-x64.iso file can be found at the following path:
C:\\Program Files (x86)\\National Instruments\\RT Images\\Utilities\\Linux RT Safemode\\<version 24.8 and above>

Reply:
Added line `Do not write the entire iso file to the USB drive.` after original text.

Seems better to have them find the path in the instructions first than read the instructions and find the path in following line.

There is now a new note under this that reiterates that only 24.8 and above system image versions' iso file can be uused; this will hopefully make the whole section under the image more prominent.